open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-sync-connectors/ranger-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/securitysync/rangerconnector/listener/GovernanceEventListener.java

```diff
@@ -5,20 +5,17 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.odpi.openmetadata.accessservices.governanceengine.api.events.GovernanceEngineEvent;
 import org.odpi.openmetadata.openconnectors.governancedaemonconnectors.securitysync.rangerconnector.processor.GovernanceEventProcessor;
-import org.odpi.openmetadata.repositoryservices.auditlog.OMRSAuditLog;
 import org.odpi.openmetadata.repositoryservices.connectors.openmetadatatopic.OpenMetadataTopicListener;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class GovernanceEventListener implements OpenMetadataTopicListener {
 
     private static final Logger log = LoggerFactory.getLogger(GovernanceEventListener.class);
-    private OMRSAuditLog auditLog;
```

Comment:
The Audit log is managed by the repository services.  If we need the Audit log in the Governance Daemons then need a change to the OMRS/admin services.

Reply:
I believe the Audit log can be useful for the Governance Daemons too. I will remove this for the moment from Governance Daemons.

Reply:
I have been looking at the admin services - ok to use Audit log - not ok to use enterprise topic connector.

Reply:
Enterprise Connector was removed in the last update.
I will keep the OMRSAuditLog.